docs/source/tutorials/base/how_to_train/classification.rst

```diff
@@ -1,7 +1,234 @@
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights models/weights.pth \
+                      --save-performance performance.json
+
+We will get a similar to this validation output:
+
+.. code-block::
+
+  ...
+
+  2023-02-03 23:43:29,514 | INFO : run task done.
+  2023-02-03 23:43:35,859 | INFO : called evaluate()
+  2023-02-03 23:43:35,870 | INFO : Accuracy after evaluation: 0.9659400544959128
+  2023-02-03 23:43:35,871 | INFO : Evaluation completed
+  Performance(score: 0.9659400544959128, dashboard: (3 metric groups))
+
+*********
+Export
+*********
+
+1. ``otx export`` exports a trained Pytorch `.pth` model to the OpenVINO™ Intermediate Representation (IR) format.
+It allows running the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run POT optimization. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+
+2. We can run the below command line to export the trained model
+and save the exported model to the ``openvino_model`` folder.
+
+.. code-block::
+
+  (otx) ...$ otx export --load-weights models/weights.pth \
+                        --save-model-to openvino_model
+
+  ...
+
+  2023-02-02 03:23:03,057 | INFO : run task done.
+  2023-02-02 03:23:03,064 | INFO : Exporting completed
+
+
+3. We can check the accuracy of the IR model and the consistency between the exported model and the PyTorch model,
+using ``otx eval`` and passing the IR model path to the ``--load-weights`` parameter.
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights openvino_model/openvino.xml \
+                      --save-performance openvino_model/performance.json
+
+  ...
+
+  Performance(score: 0.9659400544959128, dashboard: (3 metric groups))
+
+
+*************
+Optimization
+*************
+
+1. We can further optimize the model with ``otx optimize``.
+It uses NNCF or POT depending on the model format.
+
+Please, refer to :doc:`optimization explanation <../../../explanation/additional_features/models_optimization>` section to get the intuition of what we use under the hood for optimization purposes.
+
+2. Command example for optimizing
+a PyTorch model (`.pth`) with OpenVINO™ NNCF.
+
+.. code-block::
+
+  (otx) ...$ otx optimize --load-weights models/weights.pth --save-model-to nncf_model
+
+  ...
+
+  INFO:nncf:Loaded 983/983 parameters
+  2023-02-04 00:06:11,725 | INFO : run task done.
+  2023-02-04 00:06:16,924 | INFO : called evaluate()
+  2023-02-04 00:06:16,935 | INFO : Accuracy after evaluation: 0.9591280653950953
+  2023-02-04 00:06:16,936 | INFO : Evaluation completed
+  Performance(score: 0.9591280653950953, dashboard: (3 metric groups))
+
+The optimization time relies on the hardware characteristics, for example on 1 GeForce 3090 and Intel(R) Core(TM) i9-10980XE it took about 10 minutes.
```

Comment:
I like that you wrote here your processor characteristics together with GPU. I'll update it for detection as well